docs/docusaurus/docs/conceptual_guides/gx_overview.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Great Expectations overview
+---
+
+import CheckpointFlowchart from './images_overview/checkpoint_flowchart.png'
+
+This guide is intended for new users of Great Expectations (GX) and those looking for a more foundational understanding of GX, its components, and its principal workflows.  You will be introduced to the moving parts of GX and how they interact without digging through the code that governs those processes and interactions.  The insight provided in this guide will help you understand the how and why of processes you work through in future tutorials.
```

Comment:
nice, i like this

Reply:
```suggestion
The information provided here is intended for new users of Great Expectations (GX) and those looking for an understanding of its components and its primary workflows. This overview of GX doesn’t require an in-depth understanding of the code that governs GX processes and interactions. This is an ideal place to start if you plan on moving to more advanced tutorials.
```

Reply:
I've incorporated the majority of Rob's suggested edit.